src/components/CommentModal.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  Heading,
+  FormControl,
+  Input,
+  FormHelperText,
+  Spacer,
+} from "@chakra-ui/react";
+import Comment from "./Comment";
+
+const CommentModal = ({ isOpen, onOpen, onClose, post, onComment }) => {
+  const [comment, setComment] = useState("");
+  const charLeft = 250 - comment.length;
+
+  const onChange = (e) => {
+    e.preventDefault();
+    if (e.target.value.length <= 250) {
+      setComment(e.target.value);
+    }
+  };
+
+  return (
+    <>
+      <Modal
+        closeOnOverlayClick={false}
+        isOpen={isOpen}
+        onClose={onClose}
+        size={"xl"}
+      >
+        <ModalOverlay />
+        <ModalContent align={"center"}>
+          <ModalCloseButton />
+          <ModalBody pb={6}>
+            <Heading size={"xl"} color={"teal"}>{post.song_title}</Heading>
+            <Heading size={"l"}>Comments</Heading>
+            {post.comments.map((comment) =>
```

Comment:
Is there a default empty list for when there are no comments?

Reply:
yes each post is created with `[] `for comments